run_dir/static/js/running_notes.js

```diff
@@ -203,7 +202,7 @@ function count_cards(){
             }
         }
     });
-    $('.btn_count').append('All <span class="badge bg-secondary">'+all+'</span>');
```

Comment:
Shouldn't the text be at the end? 🤔

Reply:
Thats what I changed it to? The `All` is at the end.

Reply:
I know, but should it be? I thought the text is usually positioned after the class? Guess I'm wrong 😆

Reply:
If you click on Filter all of them have text after the badge. I just changed this one to make it the same as the others.

Reply:
Ah I see what you mean now!

Hmm do we want the button to have the same color as the badge on hover? I probably made the button, I don't remember :-D Or maybe change the "All"- text to white on hover so it's easier to read? 🤔 

![Skärmavbild 2023-09-26 kl  14 22 36](https://github.com/SciLifeLab/genomics-status/assets/37932118/91e18336-c8e0-4419-b5b2-dd8367fbd763)